src/encoder/encodermp3.cpp

```diff
@@ -325,33 +380,57 @@ int EncoderMp3::initEncoder(int samplerate, QString errorMessage) {
 
     m_lameFlags = lame_init();
 
-    if (m_lameFlags == NULL) {
+    if (m_lameFlags == nullptr) {
         qDebug() << "Unable to initialize MP3";
         errorMessage  = "MP3 recording is not supported. Lame could not be initialized";
         return -1;
     }
 
-    lame_set_num_channels(m_lameFlags, 2);
     lame_set_in_samplerate(m_lameFlags, samplerate_in);
     lame_set_out_samplerate(m_lameFlags, samplerate_out);
-    lame_set_brate(m_lameFlags, m_bitrate);
-    lame_set_mode(m_lameFlags, STEREO);
+        
+    lame_set_num_channels(m_lameFlags, 2);
+    // mode = 0,1,2,3 = stereo, jstereo, dual channel (not supported), mono
```

Comment:
The API is a bit confusing, so I would need to check LAME sources to ascertain what "JOINT_STEREO" is exactly.

Lame has these modes:
( http://lame.cvs.sourceforge.net/viewvc/lame/lame/doc/html/detailed.html#m . Btw, I wrote this help ;) )

> -m m	mono
> -m l	get only the left channel of a stereo signal for a mono output
> -m r	get only the right channel of a stereo signal for a mono output
> -m s	forced L/R stereo
> -m j / -m a	automatic switch between L/R and M/S stereo
> -m f	forced mid/side stereo
> -m d	dual (independent) channels.

So:
  -m m, -m l and -m r is  3 = MONO
  -m d is 2 = DUAL CHANNEL
and then, I GUESS , -m s is 0 = STEREO and -m f is 1 = JOINT STEREO.
With that suposition,  -m j / -m a is "unspecified"
The files i've created this way show joint stereo.

In LAME sources (file libmp3lame/lame.c) the default mode is "NOT_SET": 

>    gfp->mode = NOT_SET;